app.js

```diff
@@ -33,12 +33,9 @@ const resetGame = () => {
 
 const playRound = () => {
     let randomNumber = generateRandomNumber();
```

Comment:
yes! good job getting travis to pass. He's fickle with what he likes in code. ;]